containers/multisite/wp-config.php

```diff
@@ -93,14 +94,13 @@
 }
 
 /* Multisite */
-
-// define('WP_ALLOW_MULTISITE', true);
-define('MULTISITE', true);
+define('WP_ALLOW_MULTISITE', true);
```

Comment:
@ktbartholomew I have tried to use the env file and the docker compose. I notice that there was a mistake with the compose override. I was not able to connect with the env file either, but I noticed that the database hostname was docker, so I just added my ip to the hosts file.